### ISSUE TITLE
unable to reproduce the resultsof TimeGrad  on some datasets 

### ISSUE THREAD

**Issue by yantijin** [zalandoresearch/pytorch-ts#50]
I'm very glad to read the paper "Autoregressive Denoising Diffusion Models for Multivariate Probabilistic Time Series Forecasting"， it's a very intesting work and TimeGrad achieves state of the art results on multivariate time series forecasting tasks.  However, I can not reproduce the results on some datasets with the model implemented in [pytorch-ts](https://github.com/zalandoresearch/pytorch-ts). Could you release the hyperparameter settings of these datasets in the paper? Thx a lot!

**Comment by kashif** [comment]
thanks @yantijin I will put my experiment notebooks on github ASAP. Is there any dataset in particular you would like me help you with initially?

Best wishes,
Kashif

**Comment by kashif** [comment]
@yantijin awesome so next is taxi... I'm going over the notebook again, note in the meantime, a number of things have happened in the underlying gluonts library: they have refactored how the metrics are calculated and I have fixed the random seed in multi-worker training so I will re-run the notebooks and report back the hyper-params for taxi shortly

**Comment by kashif** [comment]
So for `taxi` you should try something like:

```py
estimator = TimeGradEstimator(
    target_dim=min(2000,int(dataset.metadata.feat_static_cat[0].cardinality)),
    prediction_length=dataset.metadata.prediction_length,
    context_length=dataset.metadata.prediction_length,
    cell_type='GRU',
    num_cells=128,
    input_size=2434,
    loss_type='l2',
    diff_steps=200,
    beta_end=0.15,
    freq=dataset.metadata.freq,
    scaling=True,
    trainer=Trainer(device=device,
                    epochs=50,
                    learning_rate=1e-3,
                    num_batches_per_epoch=100,
                    batch_size=32)
)
```

and perhaps even train it a bit longer...

**Comment by oumy585** [comment]
Dear Kashif,

Thank you for the article and for sharing your work.

I would like to know if I can use your method with interest rate data consisting of 660 rows and two columns (Maturity and Rate). I am doubting between timegrad and timeGAN. Thanks you for your reponse !
